Dockerfile

```diff
@@ -1,7 +1,3 @@
-FROM nginx:stable-alpine
-
-RUN rm -v /etc/nginx/conf.d/default.conf
```

Comment:
We'll overwrite this file when we run the container, so taking out this `rm` is fine. However, we should also remove `/etc/nginx/sites-enabled/default` so that it won't set up any locations that we don't need.